cmd/wallet-web/src/main.js

```diff
@@ -11,9 +11,10 @@ import VueRouter from "vue-router";
 import routes from "./router/index";
 import * as polyfill from "credential-handler-polyfill";
 import * as webCredentialHandler from "web-credential-handler";
-import MaterialDashboard from "./material-dashboard";
+//import MaterialDashboard from "./material-dashboard";
```

Comment:
why comment? why can't remove it?

Reply:
it was a typo, missed to roll back. Done now.